### PR TITLE
Add deleteComment() method

### LINKED ISSUE #53: Is it possible to delete comments?
According to [SteemConnect OAuth2](https://github.com/steemit/steemconnect/wiki/OAuth-2#scopes) there is a scope for deleting comments. It would be handy to add a method to do just that, or at least document how to do that and that it is actually supported...

Thanks!

### PR DESCRIPTION
**Pull request by tchap**
Closes #53

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -123,6 +123,15 @@ SteemConnect.prototype.comment = function comment(
...
 
+SteemConnect.prototype.deleteComment = function deleteComment(author, permlink, cb) {
+  const params = {
+    author,
+    permlink
+  };
+  return this.broadcast([['delete_comment', params]], cb);
+};
+
+
```
Comment (bonustrack) [inline src/index.js]: Here there is 2 empty lines, there should be only 1 line as separation between methods

Comment (tchap) [reply src/index.js]: Fixed.

**Comment by tchap** [comment]
@bonustrack So is there anything else I need to do to get this merged?

**Comment by bonustrack** [comment]
It's now merged @tchap, it will be released as `steemconnect@2.0.1`.

**Comment by bonustrack** [comment]
@tchap it's now released as v2.0.1. Thank you for your contribution.

**Pull request merged**